README.md

```diff
@@ -1 +1,1205 @@
+[Live]    Device state settled
+[Logs]    [9/5/2019, 2:34:41 PM] [main]
+[Logs]    [9/5/2019, 2:34:41 PM] [main] > balena-cli-masterclass@1.0.0 start /usr/src/app
+[Logs]    [9/5/2019, 2:34:41 PM] [main] > node src/helloworld.js; sleep infinity
+[Logs]    [9/5/2019, 2:34:41 PM] [main]
+[Logs]    [9/5/2019, 2:34:41 PM] [main] Hello world!
+```
+Once the code has been built on the device, it immediately starts executing
+and logs are output to the console. You can halt the connection to the local
+device by using `Ctrl-C`.
+
+In a multicontainer environment, it may quickly become difficult for an engineer
+to determine whether their code is working, especially if many services are
+all outputting log information. In these cases, filtering log output via
+service is possible, by using the `--service` switch:
+```
+$ balena push 827b231.local --service main
+[Info]    Starting build on device 827b231.local
+[Info]    Creating default composition with source: .
+[Build]   [main] Step 1/8 : FROM balenalib/fincm3-node:8
+[Build]   [main]  ---> 392c3f6339f7
+[Build]   [main] Step 2/8 : WORKDIR /usr/src/app
+[Build]   [main]  ---> Using cache
+[Build]   [main]  ---> c27a2f22ba27
+[Build]   [main] Step 3/8 : COPY package.json package-lock.json ./
+[Build]   [main]  ---> Using cache
+[Build]   [main]  ---> fc79948ab18b
+[Build]   [main] Step 4/8 : RUN npm install --ci --production     && npm cache clean --force     && rm -f /tmp/*
+[Build]   [main]  ---> Using cache
+[Build]   [main]  ---> 0cdc6d1d7af9
+[Build]   [main] Step 5/8 : COPY . ./
+[Build]   [main]  ---> 5adcd43b12c6
+[Build]   [main] Step 6/8 : CMD ["npm", "start"]
+[Build]   [main]  ---> Running in a415b6e7f0af
+[Build]   [main] Removing intermediate container a415b6e7f0af
+[Build]   [main]  ---> 1a1ff3926d42
+[Build]   [main] Step 7/8 : LABEL io.resin.local.image=1
+[Build]   [main]  ---> Running in 7390bcd46425
+[Build]   [main] Removing intermediate container 7390bcd46425
+[Build]   [main]  ---> ebf9fc9c43ed
+[Build]   [main] Step 8/8 : LABEL io.resin.local.service=main
+[Build]   [main]  ---> Running in 52ab62708a46
+[Build]   [main] Removing intermediate container 52ab62708a46
+[Build]   [main]  ---> 60f2a99b07c8
+[Build]   [main] Successfully built 60f2a99b07c8
+[Build]   [main] Successfully tagged local_image_main:latest
+
+[Live]    Waiting for device state to settle...
+[Info]    Streaming device logs...
+[Live]    Watching for file changes...
+[Logs]    [9/5/2019, 2:46:54 PM] [main]
+[Logs]    [9/5/2019, 2:46:54 PM] [main] > balena-cli-masterclass@1.0.0 start /usr/src/app
+[Logs]    [9/5/2019, 2:46:54 PM] [main] > node src/helloworld.js; sleep infinity
+[Logs]    [9/5/2019, 2:46:54 PM] [main]
+[Logs]    [9/5/2019, 2:46:54 PM] [main] Hello world!
+```
+As you can see, none of the Supervisor logs were printed.
+
+Local Mode also has another huge benefit, known as Livepush. Livepush is a
+feature of the balena Supervisor which makes intelligent decisions on how, or
```

Comment:
it's not really part of the supervisor at all, and is strictly client side, so if anything it's a CLI feature. It'd also be good to link to the repo, so if people notice something going wrong they can create an issue there.